src/Nerve_GIC/include/gudhi/GIC.h

```diff
@@ -864,7 +865,7 @@ class Cover_complex {
   template <class AssignmentRange>
   void set_cover_from_range(AssignmentRange const& assignments) {    
     std::vector<int> cov_elts, cov_number;
-    for(int i=0; i < assignments.size(); i++){
+    for(std::size_t i=0; i < assignments.size(); i++){
```

Comment:
Aren't you replacing one warning with a different one for `cover_back[cov].push_back(i);`?

Reply:
Yes you are right it was creating a warning with '-Wconversion'. Fixed on [8a3bb3d](https://github.com/GUDHI/gudhi-devel/pull/854/commits/8a3bb3dfcbb62f4aefd8175fbc7a19c72b22e38c)